pytac/lattice.py

```diff
@@ -435,17 +480,32 @@ class EpicsLattice(Lattice):
 
     Allows efficient get_values() and set_values() methods, and adds
     get_pv_names() method.
+
+    **Attributes:**
+
+    Attributes:
+        name (str): The name of the lattice.
+        symmetry (int): The symmetry of the lattice (the number of cells).
+
+    .. Private Attributes:
+           _elements (list): The list of all the element objects in the lattice
+           _cs (ControlSystem): The control system to use for the more
+                                 efficient batch getting and setting of PVs.
+           _data_source_manager (DataSourceManager): A class that manages the
+                                                      data sources associated
+                                                      with this lattice.
     """
-    def __init__(self, name, epics_cs):
+    def __init__(self, name, epics_cs, symmetry=6):
```

Comment:
I don't think we should set a default symmetry.

Reply:
added
```
    def set_lattice(self, lattice):
        """Set the stored lattice reference for this element to the passed
        lattice object.

        Args:
            lattice (Lattice): lattice object to store a reference to.
        """
        self._lattice = lattice
```